hasher-matcher-actioner/.devcontainer/devcontainer.json

```diff
@@ -19,6 +19,8 @@
 		"rvest.vs-code-prettier-eslint",
 		"ms-azuretools.vscode-docker",
 		"hashicorp.terraform",
+		"eamodio.gitlens",
+		"stkb.rewrap"
```

Comment:
Will this make our formatting non-standard for those using/not using the plugin?

Reply:
No. This is only for comment wrapping. I could not find an equivalent black command for docstrings.

This is also not enabled by default. You have to press keys [opt + q] to get this to format the comment..

Reply:
Okay, so an optionally layer for comments on top of prettier/black? works for me.